src/common/modals/Settings/components/General.js

```diff
@@ -196,6 +196,22 @@ function copy(setCopied, copyText) {
   }, 500);
 }
 
+function dashUuid(UUID) {
+  // UUID is segmented into: 8 - 4 - 4 - 4 - 12
+  // Then dashes are added between.
+
+  // eslint-disable-next-line
```

Comment:
Why disable eslint?

Reply:
oh must be left over from what i did before and i missed it. oops.

Reply:
![image](https://user-images.githubusercontent.com/8376628/77714402-f4c49680-6fa6-11ea-82a4-240014a40199.png)